src/test/java/guitests/RepositorySelectorTest.java

```diff
@@ -97,6 +98,15 @@ public void repositorySelectorTest() {
         assertEquals(4, comboBox.getItems().size());
         assertEquals("dummy4/dummy4", primaryRepo);
 
+        // we check if delete repo works
+        click("Repos");
+        push(KeyCode.DOWN);
+        push(KeyCode.RIGHT);
+        push(KeyCode.DOWN);
+        push(KeyCode.ENTER);
+        sleep(5000);
```

Comment:
This can be replaced with a combination of `PlatformEx.waitOnFxThread` and `UITest.waitForValue`/`UITest.waitUntilNodeAppears`.